### PR TITLE
Add Organization resource

### PR DESCRIPTION
**Pull request by agermain**
### Description of your changes

Adds the `grafana_organization` resource to the OSS group.

I have:

- [X] Read and followed Crossplane's [contribution process].
- [X] Run `make reviewable test` to ensure this PR is ready for review.

### How has this code been tested

Tested locally, e2e as part of this PR.

[contribution process]: https://git.io/fj2m9

### CODE REVIEW

File: config/grafana/config.go [hunk 1]
```diff
@@ -107,4 +107,11 @@ func Configure(p *ujconfig.Provider) {
 			Extractor:         SelfPackagePath + ".UserEmailExtractor()",
 		}
 	})
+	p.AddResourceConfigurator("grafana_organization", func(r *ujconfig.Resource) {
+		r.References["organization"] = ujconfig.Reference{
+			TerraformName:     "grafana_organization",
+			RefFieldName:      "OrganizationRef",
+			SelectorFieldName: "OrganizationSelector",
+		}
+	})
```
Comment (julienduchesne) [inline config/grafana/config.go]:
```suggestion
	p.AddResourceConfigurator("grafana_organization", func(r *ujconfig.Resource) {
		r.References["organization"] = ujconfig.Reference{
			TerraformName:     "grafana_organization",
			RefFieldName:      "OrganizationRef",
			SelectorFieldName: "OrganizationSelector",
		}
	})
```

This block does nothing, you can remove it. References refer to attributes of the resource. In this case, it's looking for an `organization` attribute within the `grafana_organization` resource, which does not exist

**Review by julienduchesne — COMMENTED** [review]
Nice, thanks. Just one comment

File: config/grafana/config.go (hunk 1 above)
Comment (agermain) [reply config/grafana/config.go]: Thanks!

**Review by julienduchesne — APPROVED** [review]
LGTM!

**Pull request merged**